fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -345,6 +304,41 @@ def query_categories(category_keywords: Dict[str, List[str]] = KEYWORDS) -> pd.D
     return df_raw
 
 
+# TODO: need to account for spelling variations w/in a new colum in the google sheet
+# todo: $validate-code: there are actually 2 different operations w/ this same name. might need to disambiguate
+def _get_keywords(
+    sheet_name='category_keywords', cache_path=CONFIG['keywords_cache_path'], use_cache_only=False
+) -> Dict[str, List[str]]:
+    """Get keywords from google sheets, else cache
+    sheet_name: The name of the specific sheet within a GoogleSheet workbook."""
+    # Load
+    if use_cache_only:
```

Comment:
I added a param to the `_get_keywords()` function called `use_cache_only`. If you're having any trouble with fetching from google sheets, you can also set this to `True` temporarily.